source/tests/yats-test/CMakeLists.txt

```diff
@@ -22,6 +22,7 @@ set(sources
     connection_test.cpp
     identifier_test.cpp
     main.cpp
+    parallel_sort.h
```

Comment:
This is not a test, it is an example. This m e ans you have to move your code to the excample subfolder and create an executable for it.